docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/gha-dispatch-workflow.md

```diff
@@ -12,10 +12,48 @@ description: Dispatches GitHub Actions workflows using the workflow_dispatch eve
 This promotion step is only available in Kargo on the [Akuity Platform](https://akuity.io/akuity-platform), versions v1.8 and above.
 :::
 
+:::warning Breaking Changes in v1.9
+Starting with Kargo v1.9, this promotion step has significant configuration changes:
```

Comment:
I learned that apparently you have to add newlines after/before the `:::`: https://github.com/akuity/kargo/pull/5448#discussion_r2608600474

```suggestion
:::warning Breaking Changes in v1.9

Starting with Kargo v1.9, this promotion step has significant configuration changes:
```

Reply:
updated